dataeng/jobs/createJobsNew.groovy

```diff
@@ -0,0 +1,130 @@
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.DEFAULT_VIEW
+import org.yaml.snakeyaml.Yaml
+import org.yaml.snakeyaml.error.YAMLException
+
+def globals = binding.variables
+String commonVarsDir = globals.get('COMMON_VARS_DIR')
+String commonVarsFilePath = commonVarsDir + 'common.yaml'
+Map commonConfigMap = [:]
+
+try {
+    out.println('Parsing secret YAML file')
+    String commonConfigContents = readFileFromWorkspace(commonVarsFilePath)
+    Yaml yaml = new Yaml()
+    commonConfigMap = yaml.load(commonConfigContents)
+    out.println('Successfully parsed secret YAML file')
+
+}  catch (YAMLException e) {
+    throw new IllegalArgumentException("Unable to parse ${commonVarsFilePath}: ${e.message}")
+}
+
+def taskMap = [
+    // Add jobs here as they are ported from the old analytics Jenkins server.
```

Comment:
Let me know if this is out of scope, but i was wondering if we could take this opportunity to make this new seed job more like a conventional seed job that only seeds one file at a time.  Would that require a major overhaul of how we write our job files?

Reply:
The reason I ask is because your subteam was spun off for quality of life improvements, and being able to simply test a job DSL without having to create a temp commit to comment out this taskMap would increase my quality of life.

Reply:
I would like to do this, but we should think it through. For the sake of what I am doing right now (trying to get a box up asap to run warehouse transform ci), I would consider it out of scope, but please put in a ticket so we can circle back to address it.